### PR TITLE
[LUPEYALPHA-954] FE admin tasks

### PR DESCRIPTION
**Pull request by asmega**
# Context

- https://dfedigital.atlassian.net/browse/LUPEYALPHA-954
- FE admin tasks
- There are placeholder present to unblock areas of work for others

### CODE REVIEW

File: app/models/automated_checks/claim_verifiers/employment.rb [hunk 1]
```diff
@@ -11,13 +11,18 @@ def initialize(claim:, admin_user: nil)
...
+        return unless required?
         return unless awaiting_task?
 
         no_data || no_match || matched
       end
 
       private
 
+      def required?
+        claim.eligibility.teacher_reference_number.present?
```
Comment (asmega) [inline app/models/automated_checks/claim_verifiers/employment.rb:23]: not sure if this negatively impacts other policies?

Comment (rjlynch) [reply app/models/automated_checks/claim_verifiers/employment.rb:23]: I think this should be ok, most of the other journeys require a TRN from the claimant to submit, IRP doesn't but it doesn't use this claim verifier

File: app/models/claim_checking_tasks.rb [hunk 2]
```diff
@@ -12,26 +12,34 @@ def initialize(claim)
...
-        task_names.delete("arrival_date")
-        task_names.delete("employment_contract")
-        task_names.delete("employment_start")
-        task_names.delete("subject")
-        task_names.delete("teaching_hours")
-      end
-      unless claim.policy.further_education_payments?
+    case policy
+    when Policies::FurtherEducationPayments
+      Policies::FurtherEducationPayments::ClaimCheckingTasks
```
Comment (rjlynch) [inline app/models/claim_checking_tasks.rb:17]: 👍 nice

**Review by rjlynch — APPROVED** [review]
Nice 👍
I think pulling out a class for the policies task lists is a great idea

**Pull request merged**